### ISSUE TITLE
Support for build 2024.1

### ISSUE THREAD

**Issue by ZeroBrushV2** [evgeniy-polyakov/jsfl-support#7]
Hi, since the 2024.1 has been rolled out, the plugin is not longer working cause it's built using the version 233, but now is 241+.

Could you please update it to the latest build versione please?

**Comment by evgeniy-polyakov** [comment]
Yes, no problem. Will do today

**Comment by evgeniy-polyakov** [comment]
The plugin is updated to version 4.4.1 compatible with IntelliJ platform 2024.1: https://plugins.jetbrains.com/plugin/7096-jsfl-support/versions/stable/519253

**Comment by ZeroBrushV2** [comment]
> The plugin is updated to version 4.4.1 compatible with IntelliJ platform 2024.1: https://plugins.jetbrains.com/plugin/7096-jsfl-support/versions/stable/519253

Thank you so much! I really appreciate your plugin, keep the good work

**Comment by ZeroBrushV2** [comment]
Hi, just update it, and this error shown from intellij:

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'org.jsflsupport.actions.RunJSFLFileAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: JSFL Support]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)

```

**Comment by evgeniy-polyakov** [comment]
I cannot reproduce this issue with fresh install of the plugin. Is that a deprecation warning? If so it should allow the plugin to work anyway. I will check deprecated methods later.

**Comment by ZeroBrushV2** [comment]
> I cannot reproduce this issue with fresh install of the plugin. Is that a deprecation warning? If so it should allow the plugin to work anyway. I will check deprecated methods later.

The error shows up in bottom right corner of the IDE when I do right click inside a class/file, telling me there was an exception on the plugin, if I open up the windows it tells me this:
![image](https://github.com/evgeniy-polyakov/jsfl-support/assets/123548294/bf9243e6-5c65-41b3-8065-ea5f2b7ef156)

The options like "Create JSFL File" still works, I think maybe has something to do with the validation?! I can't figure out what exactly is wrong.

Anyway don't rush since is not urgent, I'm not currently have necessity to use the Plugin in the current project

**Comment by evgeniy-polyakov** [comment]
The warning is fixed in version [4.4.2](https://plugins.jetbrains.com/plugin/7096-jsfl-support/versions/stable/532327)
